vassal-app/src/main/java/VASSAL/build/module/GlobalOptions.java

```diff
@@ -498,7 +504,20 @@ public boolean autoReportEnabled() {
   public boolean centerOnOpponentsMove() {
     return Boolean.TRUE.equals(GameModule.getGameModule().getPrefs().getValue(CENTER_ON_MOVE));
   }
-  
+
+  public double centerOnOpponentsMoveSensitivity() {
+    Integer intSensitivity = (int) GameModule.getGameModule().getPrefs().getValue(CENTER_ON_MOVE_SENSITIVITY);
+    double sensitivity;
```

Comment:
You don't need to turn this into a double yourself; you can just return the int and it will be upcast.

Reply:
I think what you want here is:
```
int sensitivity = (Integer) GameModule.getGameModule().getPrefs().getValue(CENTER_ON_MOVE_SENSITIVITY);
```